vera/README.md

```
@@ -80,7 +80,7 @@ execute(compile_result, ...)    # → run WASM via wasmtime
 | `  core.py` | 952 | | TypeChecker class, orchestration, contracts, constraint validation | |
 | `  resolution.py` | 419 | | AST TypeExpr → semantic Type, inference | |
 | `  modules.py` | 180 | | Cross-module registration (C7b/C7c) | |
-| `  registration.py` | 456 | | Pass 1 forward declarations, ability registration | |
+| `  registration.py` | 528 | | Pass 1 forward declarations, ability registration | |
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)